pages/api/resolvers/resolvers.ts

```diff
@@ -31,17 +37,25 @@ export const resolvers = {
       return question;
     },
   },
-  Question: {
+  Question: { //overwrite field resolver
+    correctAnswer(question: any){
```

Comment:
```suggestion
    correctAnswer(question: QuestionDbModel){
```

We can add the typing now can't we?

Same goes for line 44